modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -637,7 +637,8 @@ protected String getVariableTypeName(
 	}
 
 	protected boolean hasClassOrVariableName(
-		String className, String content, String methodCall) {
+		String className, String content, String fileContent,
+		String methodCall) {
```

Comment:
@TamyrisTorres, should we just include this new parameter or does make sense we keep the method as it's, I mean, with 3 parameters, and create a new one? What do you think about it?